[2022-01-11 20:17:53,146] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_01 scheduled__2022-01-06T00:00:00+00:00 [queued]>
[2022-01-11 20:17:53,255] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_01 scheduled__2022-01-06T00:00:00+00:00 [queued]>
[2022-01-11 20:17:53,259] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-11 20:17:53,259] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-11 20:17:53,259] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-11 20:17:53,361] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): task_01> on 2022-01-06 00:00:00+00:00
[2022-01-11 20:17:53,392] {standard_task_runner.py:52} INFO - Started process 4723 to run task
[2022-01-11 20:17:53,455] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_01', 'scheduled__2022-01-06T00:00:00+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dags.py', '--cfg-path', '/tmp/tmpnprz98fi', '--error-file', '/tmp/tmplvvv0zgb']
[2022-01-11 20:17:53,471] {standard_task_runner.py:77} INFO - Job 41: Subtask task_01
[2022-01-11 20:17:54,229] {logging_mixin.py:109} INFO - Running <TaskInstance: parallel_dag.task_01 scheduled__2022-01-06T00:00:00+00:00 [running]> on host airflowvm
[2022-01-11 20:17:54,984] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_01
AIRFLOW_CTX_EXECUTION_DATE=2022-01-06T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-06T00:00:00+00:00
[2022-01-11 20:17:54,986] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-11 20:17:55,003] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-01-11 20:17:55,070] {subprocess.py:85} INFO - Output:
[2022-01-11 20:17:58,121] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-11 20:17:58,640] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_01, execution_date=20220106T000000, start_date=20220111T201753, end_date=20220111T201758
[2022-01-11 20:18:04,108] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-11 20:18:04,918] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
